src/plugins/intel_npu/src/plugin/npuw/partitioning/online/compiler.cpp

```diff
@@ -142,7 +142,8 @@ std::vector<Isolate> getIsolates(const std::string isolates_unparsed) {
     } else {
         LOG_WARN("Incorect pattern in NPUW_ONLINE_ISOLATE!"
                  << " Please, follow the example: "
-                 << "Op:Select/NPU,P:DQMatMulGQ/compute,P:DQMatMulCW/compute,P:RMSNorm/compute. "
+                 << "Op:Select/NPU,P:DQMatMulGQu4/compute,P:DQMatMulCWu4/compute,P:DQMatMulGQi4/compute,P:DQMatMulCWi4/"
+                    "compute,P:RMSNorm/compute. "
                  << "No isolate rules will be taken into account during partitioning!");
```

Comment:
I don't think it make sense to enumerate them all here. Just write that the pattern is wrong.

Reply:
Removed